maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -267,6 +267,9 @@ class Orchestra(
 
     private fun clearAppStateCommand(command: ClearStateCommand): Boolean {
         maestro.clearAppState(command.appId)
+        // Android's clear command also resets permissions
+        // Reset all permissions to unset so both platforms behave the same
+        maestro.setPermissions(command.appId, mapOf("all" to "unset"))
```

Comment:
On Android call to clearAppState will bring back the permissions to unset case by default. Isn't that the for iOS?

Reply:
Though it is a no-change for Android; it's good to be explicit about what the expected result is by setting it for both platforms instead of relying on platform specific behaviours. If the setPermission call takes a long time on Android, then we better only call it on iOS.